oarepo_ui/templates/oarepo_ui/detail.html

```diff
@@ -3,29 +3,32 @@
 {%- endblock bypasslinks %}
 {%- block page_body %}
 
-<div class="ui main container">
-  <div class="ui relaxed grid">
-    {% block record_header %}
-    {% endblock %}
-    <div class="two column row">
-      {# Main content #}
-      <article
-        class="sixteen wide tablet twelve wide computer column main-record-content"
-      >
-        {% block record_main_content %}
-        {% endblock %}
-      </article>
+  <PreviewHeader is_preview={is_preview} />
```

Comment:
I would lift `is_preview` up to the detail page, and render whole PreviewHeader conditionally